htdocs/install/mysql/tables/llx_bookcal_booking-bookcal.sql

```diff
@@ -35,6 +35,7 @@ CREATE TABLE llx_bookcal_booking(
 	lastname varchar(128) NOT NULL, 
 	email varchar(128) NOT NULL, 
 	start datetime NOT NULL, 
-	duration integer NOT NULL
+	duration integer NOT NULL, 
+	fk_bookcal_availability integer NOT NULL
```

Comment:
This field is an id of another table ? Foregeign key of each table ?

For me bookcal_bookeeping is table with list of all calendars for example "Calendar for commercial contact". So it is a parent table.
Then inside a calendar, there is n entries of availabilities so table llx_boockcal_availability

Reply:
As the module is created it's the contrary.
llx_boockcal_availabilities is the table that is a list of all the calendars and llx_bookcal_booking is a table that permit to create appointment with someone

So fk_bookcal_availability is the link between the appointment and the calendar id present in llx_boockcal_availabilities